src/person/controllers/update_person_controller.py

```diff
@@ -15,6 +15,8 @@
 
 from ..tasks import upload_profile_pic
 
```

Comment:
delete line

Reply:
shouldn't there be pre-commit formatter being used?